.buildkite/pipeline.yml

```diff
@@ -1,3 +1,88 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: 'v0.5.0'
+  DOCKER_COMPOSE_VERSION: '1.25.5'
+  DOCKER_REGISTRY: "docker.elastic.co"
+  DOCKER_IMAGE: "${DOCKER_REGISTRY}/observability-ci/temp-fleet-server" # needs to rename after tests
+  DOCKER_IMAGE_TAG: ""
+  DOCKER_IMAGE_SHA_TAG: "temp-${BUILDKITE_COMMIT}" # needs to rename after tests
+  DOCKER_IMAGE_LATEST_TAG: "temp-latest" # needs to rename after tests
+  DOCKER_IMAGE_GIT_TAG: "temp-${BUILDKITE_BRANCH}" # needs to rename after tests
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - group: "Check and build"
+    key: "check"
+    steps:
+      - label: ":go: Run check-ci"
+        key: check-ci
+        command: "make check-ci"
+        agents:
+          image: "golang:1.20.3"
```

Comment:
This applies to the rest of the references about the golang version. 
Ideally having this be sourced from the `.go-version` would be ideal, alternative it should be defined as env var